pipes/WDL/workflows/assemble_refbased.wdl

```diff
@@ -30,10 +31,6 @@ workflow assemble_refbased {
         aligner: {
             description: "Read aligner software to use. Options: novoalign, bwa, minimap2. Minimap2 can automatically handle Illumina, PacBio, or Oxford Nanopore reads as long as the 'PL' field in the BAM read group header is set properly (novoalign and bwa are Illumina-only)."
         }
-        novocraft_license: {
```

Comment:
While we don't use this much these days, this is still an optional parameter in the workflow and we should keep the documentation for it